proposals/variant_set_metadata/example.usda

```diff
@@ -0,0 +1,62 @@
+#usda 1.0
+(
+    defaultPrim = "MilkCartonA"
+)
+
+def Xform "MilkCartonA" (
+    kind = "prop"
+    variants = {
+        string modelingVariant = "Carton_Opened"
+        string shadingComplexity = "full"
+        string shadingTest = "milkBrandA"
+    }
+    add variantSets = ["modelingVariant", "shadingComplexity", "shadingTest"]
```

Comment:
Same note as above.  `append` is the preferred replacement for `add`.

Reply:
Updated to `prepend`